src/main/java/seedu/address/model/employee/Employee.java

```diff
@@ -101,8 +101,7 @@ public boolean equals(Object other) {
                 && phone.equals(otherEmployee.phone)
                 && email.equals(otherEmployee.email)
                 && address.equals(otherEmployee.address)
-                && tags.equals(otherEmployee.tags)
-                && project.equals(otherEmployee.project);
```

Comment:
project.equals is removed because the projects of employees are constantly updated and it would be impractical to update every instance of said employee in the projects involved.

Reply:
Are you referring to the multiple instances of the same employee that exists across multiple projects?

Reply:
yes